AWS/legos/aws_get_service_quota_detailsv1/aws_get_service_quota_detailsv1.py

```diff
@@ -0,0 +1,46 @@
+
+from __future__ import annotations
+
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+
+class InputSchema(BaseModel):
+    quota_code: Optional[str] = Field(
```

Comment:
@dougsillars shouldnt service_code and quota_code be an enum?

Reply:
no. There are thousands of quota codes, and new ones added on the regular.  If we force enum values, our Action will be out of date as soon as we publish it,

Reply:
How easy is it for someone to get these quota codes? Is it on webpage somewhere? We should put that link in the description then. @dougsillars